packages/common/src/dbt/schemas/lightdashV12.json

```diff
@@ -0,0 +1,64 @@
+{
+    "definitions": {
+        "LightdashNodeConfig": {
+            "allOf": [
+                {
```

Comment:
Notice that here we usually extend the dbt definition. 

They have changed the manifest json schema `https://schemas.getdbt.com/dbt/manifest/v12.json` to use draft 2020-12 `https://json-schema.org/draft/2020-12/schema` format.

I couldn't find a way to extend these definitions. 
Let me know if there is a way. 🙏